frontend/src/store/slice/AlgorithmNode/AlgorithmNodeSlice.ts

```diff
@@ -107,7 +110,7 @@ export const algorithmNodeSlice = createSlice({
           Object.values(runPostData.nodeDict)
             .filter(isAlgorithmNodePostData)
             .forEach((node) => {
-              state[node.id].isUpdated = false
+              state[node.id].originalValue = false
```

Comment:
There appears to be a mix of cases for using `originalValue`, right?
- as a boolean value that show if the parameter is different from the original value
- too keep the original value

Please focus on the one case (may be the latter. the former would be the role of selector.), as it may cause confusion for other engineers.

Reply:
@ReiHashimoto I have corrected it, please check to see if it is correct